src/components/CelebrityInfoBox.js

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from 'react';
+import Loader from './Loader/Loader';
+import styles from './CelebrityInfoBox.module.css';
+
+const CelebrityInfoBox = ({ celebrity, toggleIsLoading, isLoading, showInfoBox }) => {
+
+	const setOnload = useCallback(node => {
+		if (node !== null) {
+
```

Comment:
Here, I am calling `toggleIsLoading` to update state  of  `<App/>` after image was loaded. 
Should I add `toggleIsLoading`  to depency array of  useCallback to get rid of warning?